README.md

```diff
@@ -42,11 +57,22 @@ var Panda = function(name, age) {
   this.name = name;
   this.age = age;
   this.num_bamboo_eaten = 0;
+  this.eat_bamboo = function() {
+    this.num_bamboo_eaten++
+  }
 }
```

Comment:
:+1: This could work fine! but think about moving methods out into their own prototype declarations:
```Panda.prototype.eat_bamboo = function(){
  this.num_bamboo_eaten += 1;
}```